FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -347,7 +347,7 @@ export function SignUpFormContentComponent(props) {
                 />
                 <label className={styles['rules__line--text']}>
                   Погоджуюсь з{' '}
-                  <a href="#" onClick={openModal} className={styles['rules__line--link']}>
+                  <a onClick={openModal} className={styles['rules__line--link']}>
```

Comment:
We use Link, not a component

Reply:
const RulesModal = React.lazy(() => import('./RulesModal')); This solution allows optimizing the website and reducing loading time. There was no requirement to create a physical page; it exclusively involves dynamically changing the page

Reply:
In the styles, it's a traditional link, so I use <a>